recipe/build.sh

```diff
@@ -28,18 +28,19 @@ then
     export HOROVOD_CUDA_INCLUDE=/usr/include
 fi
 
-HOROVOD_WITHOUT_MXNET=1 HOROVOD_WITHOUT_GLOO=1 HOROVOD_WITH_PYTORCH=1 HOROVOD_WITH_TENSORFLOW=1 HOROVOD_CUDA_HOME=$CUDA_HOME HOROVOD_GPU_ALLREDUCE=NCCL HOROVOD_GPU_BROADCAST=NCCL python setup.py install --prefix=$PREFIX
+MAKEFLAGS="-j1" HOROVOD_WITHOUT_MXNET=1 HOROVOD_WITHOUT_GLOO=1 HOROVOD_WITH_PYTORCH=1 HOROVOD_WITH_TENSORFLOW=1 HOROVOD_CUDA_HOME=$CUDA_HOME HOROVOD_GPU_ALLREDUCE=NCCL HOROVOD_GPU_BROADCAST=NCCL python setup.py install --prefix=$PREFIX
```

Comment:
Does this work with a value greater than 1 for j? This is disabling parallel builds.

Reply:
Parallel builds fail with file permissions.
On a horovod issue this is what Travis recommended.